pkgs/development/libraries/qtstyleplugin-kvantum/default.nix

```
@@ -41,10 +43,11 @@ stdenv.mkDerivation rec {
     qtsvg
     libX11
     libXext
-  ] ++ lib.optionals (!isQt6) [ qtx11extras kwindowsystem ]
-    ++ lib.optional isQt6 qtwayland;
+    kwindowsystem
+  ] ++ lib.optional isQt5 qtx11extras
+    ++ lib.optional (!isQt5) qtwayland;
```

Let's use `optionals` here.

It's true, it's opinionated. But `optional` vs `optionals` is always an issue for newcomers.

`optional` implicitly wrap the value in a list, while `optionals` doesn't. `optionals` have a lower cognitive load than `optional`, and I think that we should use it here.


```suggestion
  ] ++ lib.optionals isQt5 [ qtx11extras ]
    ++ lib.optionals (!isQt5) [ qtwayland ];
```